### DISCUSSION TITLE
blogs/Tech/beyond-tech

### DISCUSSION THREAD

**Discussion by giscus** [Dustella/CandyStore#8] [General]
# blogs/Tech/beyond-tech

Dustella 的自留地

https://www.dustella.net/blogs/Tech/beyond-tech.html

**Comment by qwertycxz** [comment]
在自己的 PC 上配工具链确实不是什么简单的事情！之前在 Windows PC 上配 GCC + VSCode 开发，可给自己恶心坏了 -_-

**Comment by Dilettante258** [comment]
写的真好，狠狠赞了。♥️
